front/app/components/EsriMap/index.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useEffect } from 'react';
```

Comment:
I love the structure of this - a general map component that can then be contextualized depending on what we need.